airline-data/src/main/scala/com/patson/model/airplane/Model.scala

```diff
@@ -81,15 +81,21 @@ object Model {
                       Model("Boeing 737-700C", capacity = 140, fuelBurn = (140 * 3.3).toInt, speed = 825, range = 6083, price = 85000000, lifespan = 35 * 52, constructionTime = 12, countryCode = "US", imageUrl = "https://www.norebbo.com/2014/04/boeing-737-700-blank-illustration-templates/"),
+                      Model("McDonnel Douglas MD-90", capacity = 160, fuelBurn = (160 * 3.55).toInt, speed = 811, range = 3787, price = 80000000, lifespan = 30 * 52, constructionTime = 8, countryCode = "US", imageUrl = "https://www.norebbo.com/2018/02/mcdonnell-douglas-md-90-blank-illustration-templates/"),
                       Model("Boeing 737-800", capacity = 184, fuelBurn = (184 * 3.8).toInt, speed = 825, range = 5436, price = 100000000, lifespan = 35 * 52, constructionTime = 24, countryCode = "US", imageUrl = "https://www.norebbo.com/2012/11/boeing-737-800-blank-illustration-templates/"),
                       Model("Airbus A320neo", capacity = 195, fuelBurn = (195 * 4).toInt, speed = 833, range = 6500, price = 110000000, lifespan = 35 * 52, constructionTime = 8, countryCode = "NL", imageUrl = "https://www.norebbo.com/2017/08/airbus-a320-neo-blank-illustration-templates/"),
+                      Model("Boeing 757-200", capacity = 200, fuelBurn = (200 * 4.25).toInt, speed = 854, range = 7250, price = 95000000, lifespan = 35 * 52, constructionTime = 8, countryCode = "US", imageUrl = "https://www.norebbo.com/2015/01/boeing-757-200-blank-illustration-templates/"),
                       Model("Tupolev Tu-204", capacity = 210, fuelBurn = (210 * 4.5).toInt, speed = 810, range = 4300, price = 50000000, lifespan = 25 * 52, constructionTime = 24, countryCode = "RU"),
                       Model("Boeing 737 MAX 9", capacity = 230, fuelBurn = (230 * 4.2).toInt, speed = 839, range = 6570, price = 124000000, lifespan = 35 * 52, constructionTime = 36, countryCode = "US", imageUrl = "https://www.norebbo.com/2018/05/boeing-737-9-max-blank-illustration-templates/"),
                       Model("Boeing 787-8 Dreamliner", capacity = 250, fuelBurn = (250 * 4.5).toInt, speed = 907, range = 13621, price = 125000000, lifespan = 35 * 52, constructionTime = 36, countryCode = "US", imageUrl = "https://www.norebbo.com/2013/02/boeing-787-8-blank-illustration-templates/"),
+                      Model("Airbus A300-600", capacity = 266, fuelBurn = (266 * 4.85).toInt, speed = 833, range = 7500, price = 110000000, lifespan = 30 * 52, constructionTime = 36, countryCode = "NL", imageUrl = "https://www.norebbo.com/2018/11/airbus-a300b4-600r-blank-illustration-templates-with-general-electric-engines/"),
+                      Model("McDonnel Douglas MD-11", capacity = 293, fuelBurn = (293 * 4.8).toInt, speed = 876, range = 12670, price = 105000000, lifespan = 30 * 52, constructionTime = 36, countryCode = "US", imageUrl = "https://www.norebbo.com/2018/05/mcdonnell-douglas-md-11-blank-illustration-templates-with-ge-engines/"),
```

Comment:
MD-11 seems to be too powerful here compared to A300-600, cheaper, fly faster, burn less fuel, longer range, higher capacity

I would think MD should be a bit "inferior" in quality compared to Airbus and Boeing. 

Can i suggest that:
A300-600
- Improve fuel efficiency to maybe around 4.6 or 4.7
- Increase lifespan to 35 - all Airbus and Boeing are 35

Reply:
I think that if we did that, the A300-600 would become too good, on par with the Dreamliner, and the A300 is >20years older. 
It would be almost as efficient, but cheaper and with higher capacity.

What if we:

- Improve the A300 efficiency to 4.75
- Increase the A300 lifespan to 35
- Increase the MD-11 price by $35mi to $140mi.

Or we could just remove the A300 alltogether, maybe it's too old to balance properly.

Reply:
- The least I can go is 4.72.
JK, let's go with 4.7 then. I was afraid that you wouldn't like an older plane close to the Dreamliner.

- I haven't thought of that, yeah, that would be best.

Reply:
hehehe ya...i am running the test case with the existing parameters. the several models around that range has profit factor of:

Boeing 787-8 Dreamliner => 0.2188181083265966
Airbus A300-600 => 0.17629267343115068
McDonnel Douglas MD-11 => 0.19297209752759759
Ilyushin Il-96-300 => 0.19682969549986526

Which shows that Airbus A300-600 is heavily disadvantaged @ 0.17629267343115068

I need to head out now...will look at it again after i come back early afternoon :)

Reply:
Ok, I've made the changes you described.
Now the results are:
Boeing 787-8 Dreamliner => 0.2188181083265966
Airbus A300-600 => 0.20037837311725917
McDonnel Douglas MD-11 => 0.18553364547609127
Ilyushin Il-96-300 => 0.19682969549986526